src/FamilyHubs.ReferralUi.Ui/Pages/Shared/_Header.cshtml

```diff
@@ -20,20 +20,6 @@
                 </a>
             </div>
             <div class="govuk-header__content">
-                <nav class="govuk-header__navigation govuk-header__navigation--end" aria-label="Menu">
-                    <button type="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide menu" aria-expanded="false" hidden="">Menu</button>
-                    <ul id="navigation" class="govuk-header__navigation-list">
-                        @if (User != null && User.Identity != null && User.Identity.IsAuthenticated)
-                        {
-                        <li class="govuk-header__navigation-item">
-                                @Html.Raw(Model.RenderLink<HomeLink>())
-                        </li>
-                        <li class="govuk-header__navigation-item">
-                                @Html.Raw(Model.RenderLink<SignOutLink>())
-                        </li>
-                        }
-                    </ul>
-                </nav>
```

Comment:
If this is not required for MVP then wrap it in IsReferralEnabled

Reply:
Hi Martin we don't require this menu functionality this comes with gov.uk standard header .I think this added as part of it

Reply:
I have emailed you a screen shot showing the headers when a VCS Admin or Professional are logged in, please confirm removing this code will not break existing code.